models/ed/R/write.configs.ed.R

```diff
@@ -382,12 +382,15 @@ write.config.xml.ED2 <- function(settings, trait.values, defaults=settings$const
 
       ## Convert trait values to ED units
       converted.trait.values <- convert.samples.ED(trait.values[[i]])
-
+      
       ## Selectively replace defaults with trait values
       vals <- modifyList(vals, converted.trait.values)
 
+      ## Convert settings constants to ED units
+      converted.defaults <- convert.samples.ED(defaults[[pft]])
```

Comment:
Good call. I figured that defaults would have PFT names as list attributes already, rather than as `defaults$pft$name`, but if that's the standard, we can stick with it.